repos/system_upgrade/common/actors/scanfilestobecopiedintotargetuserspace/actor.py

```diff
@@ -0,0 +1,18 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.scanfilestobecopiedintotargetuserspace import scan_files_to_copy
+from leapp.models import CopyFile
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class ScanFilesToBeCopiedIntoTargetUserspace(Actor):
```

Comment:
What about `RequestFilesInContainer` (@mreznik @vinzenz any ideas about shorter good enougn name?)

Reply:
Maybe `PropagateFilesIntoContainer` but if we just care about shorter version here, I am indifferent.

Reply:
Discussed, we will go with the `RequestFilesInContainer`

Reply:
@MichalHe  so the final one is: `ScanFilesForTargetUserspace`

Reply:
Thanks for the tips, appreciate it.
Requested changes implemented in the latest version.

Reply:
The suggested name is used in the latest version. Thanks for the help.